app/data_definitions/from_cascade/shared_content_degree_program.html

```diff
@@ -0,0 +1,108 @@
+<system-data-structure>
```

Comment:
For sake of consistency, these new files should probably have an `.xml` extension.

Reply:
good catch. I've changed the extensions